kiwi_lint/db_columns.py

```diff
@@ -0,0 +1,21 @@
+from pylint import interfaces, checkers
+from pylint.checkers import utils
+
+
+class DbColumnChecker(checkers.BaseChecker):
+    __implements__ = (interfaces.IAstroidChecker, )
+
+    name = 'db-column-checker'
+
+    msgs = {
+        'E4481': (
+            'Avoid using db_column in Model fields',
+            'avoid-db-column'
+        )
+    }
+
+    @utils.check_messages('avoid-db-column')
+    def visit_attribute(self, node):
+        if (node.attrname == 'CharField'
+                and isinstance(node.expr, astroid.Attribute) and node.expr.attrname == 'models'):
```

Comment:
still incorrect.

Reply:
Do we use the `node.func.as_string().endswith('Field')` to check for the node?

Reply:
All field types can have db_column attribute, not only CharField. Try pylint_django.utils.node_is_subclass

Reply:
This is still unresolved.